client/blocks/comment-detail/index.jsx

```diff
@@ -190,6 +199,17 @@ export class CommentDetail extends Component {
 		}
 	}
 
+	trackDeepReaderLinkClick = () => {
+		const { isJetpack, parentCommentContent } = this.props;
```

Comment:
any reason to destructure just these two props?

Reply:
I tend to do this for 2 or more, no hard reasoning.

Reply:
I thought there was a new rule I wasn't aware of, like, destructure variables but not functions 😌